<ui version="4.0" >
 <class>JabberAdd</class>
 <widget class="QWidget" name="JabberAdd" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="RadioGroup" native="1" name="grpJID" >
     <property name="title" stdset="0" >
      <string>&amp;JID</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>11</number>
      </property>
      <item>
       <widget class="QLineEdit" name="edtJID" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="RadioGroup" native="1" name="grpMail" >
     <property name="title" stdset="0" >
      <string>&amp;E-Mail</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>11</number>
      </property>
      <item>
       <widget class="QLineEdit" name="edtMail" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="RadioGroup" native="1" name="grpName" >
     <property name="title" stdset="0" >
      <string>&amp;Name</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>11</number>
      </property>
      <item>
       <widget class="QLabel" name="lblFirst" >
        <property name="text" >
         <string>First:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtFirst" />
      </item>
      <item>
       <widget class="QLabel" name="lblLast" >
        <property name="text" >
         <string>Last:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtLast" />
      </item>
      <item>
       <widget class="QLabel" name="lblNick" >
        <property name="text" >
         <string>Nickname:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtNick" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="Spacer2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnBrowser" >
     <property name="text" >
      <string>Jabber &amp;browser</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
    <customwidget>
        <class>RadioGroup</class>
        <header location="local">simgui/intedit.h</header>
    </customwidget>
</customwidgets>
<resources/>
 <connections/>
</ui>
